semantic_ai/nlp/prompt.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+from langchain.chains import LLMChain
+from langchain.prompts import PromptTemplate
+from langchain.utilities import SQLDatabase
+from langchain_experimental.sql import SQLDatabaseChain
+
+from semantic_ai.llm import Openai
+from semantic_ai.utils import sync_to_async
+
+logger = logging.getLogger(__name__)
+
+SQL_DEFAULT_TEMPLATE = """Given an input question, first create a syntactically correct {dialect} working query to run,
+ then look at the results of the query and return the response from the DATABASE and give me as JSON response.
+ Do not generate response from outside of the context.	
+Use the following json format:
+
+Question: "Question here",
+SQLQuery: "SQL Query to run",
+Answer: "Final answer here"
+
+
+Only use the following tables:
+
+{table_info}
+
+If someone asks for the table, they really mean the table.
+
+Question: {input}
+
+JSON Format:
+- Question: [Input here]
+- SQLQuery: [Generated Query]
+
+Provide your SQLQuery and Answer in JSON format. Reply with only the answer in JSON format and include no other commentary:"""
+
+
+class Prompt:
+
+    def __init__(self):
+        self.prompt = PromptTemplate(
+            input_variables=["input", "table_info", "dialect"],
+            template=SQL_DEFAULT_TEMPLATE
+        )
+        self.openai_llm = Openai(model_name_or_path="gpt-3.5-turbo-1106")
```

Comment:
Why this hard coded here?!

Reply:
This is the default model. Am changed into the as a params  for model. We can change model thourgh params.